> Also, I'm curious what the issue is with atomics in miniaudio

Here's some relevant build output:

```
Undefined symbols:
  "___sync_val_compare_and_swap_8", referenced from:
      _c89atomic_compare_exchange_strong_explicit_64 in libclownaudio.a(miniaudio.c.o)
      _c89atomic_compare_exchange_strong_explicit_64 in libclownaudio.a(miniaudio.c.o)
      _c89atomic_compare_exchange_strong_explicit_64 in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_post in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_post in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_post in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_post in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_post in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_post in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_next in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_next in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_next in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_next in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_next in libclownaudio.a(miniaudio.c.o)
      _ma_job_queue_next in libclownaudio.a(miniaudio.c.o)
ld: symbol(s) not found
collect2: ld returned 1 exit status
```

I'm not sure exactly what the issue is, it could be that PowerPC doesn't support these operations, or that my linker setting are somehow wrong.

--- a/src/playback/coreaudio.c
+++ b/src/playback/coreaudio.c
@@ -29,7 +29,7 @@ PERFORMANCE OF THIS SOFTWARE.
 #include <AvailabilityMacros.h>
 
 #if !defined(MAC_OS_X_VERSION_10_6) || MAC_OS_X_VERSION_MIN_REQUIRED < MAC_OS_X_VERSION_10_6
-// These deprecated Carbon APIs were replaced with equivalent functions and types in macOS 10.6
+/* These deprecated Carbon APIs were replaced with equivalent functions and types in macOS 10.6 */
 #include <CoreServices/CoreServices.h>
 #define AudioComponent Component
 #define AudioComponentDescription ComponentDescription
@@ -58,7 +58,7 @@ static OSStatus Callback(void *inRefCon, AudioUnitRenderActionFlags *ioActionFla
 
 	ClownAudio_Stream *stream = (ClownAudio_Stream*)inRefCon;
 
-	// Because the stream is interleaved, there is only one buffer
+	/* Because the stream is interleaved, there is only one buffer */
 	stream->user_callback(stream->user_data, (short*)ioData->mBuffers[0].mData, inNumberFrames);
 
 	return 0;
@@ -66,7 +66,7 @@ static OSStatus Callback(void *inRefCon, AudioUnitRenderActionFlags *ioActionFla
 
 CLOWNAUDIO_EXPORT bool ClownAudio_InitPlayback(void)
 {
-	// Find the default output AudioUnit
+	/* Find the default output AudioUnit */
 	AudioComponentDescription default_output_description;
 	bool success = true;
 
@@ -97,12 +97,12 @@ CLOWNAUDIO_EXPORT ClownAudio_Stream* ClownAudio_StreamCreate(unsigned long *samp
 	{
 		if (default_output_component != NULL)
 		{
-			// Create a new instance of the default output AudioUnit
+			/* Create a new instance of the default output AudioUnit */
 			OSStatus error = AudioComponentInstanceNew(default_output_component, &stream->audio_unit);
 
 			if (!error)
 			{
-				// Use a suitable output format
+				/* Use a suitable output format */
 				AudioStreamBasicDescription want;
 				want.mFormatID = kAudioFormatLinearPCM;
 				want.mFormatFlags = kLinearPCMFormatFlagIsSignedInteger | kLinearPCMFormatFlagIsPacked
@@ -110,34 +110,34 @@ CLOWNAUDIO_EXPORT ClownAudio_Stream* ClownAudio_StreamCreate(unsigned long *samp
 				                    | kAudioFormatFlagIsBigEndian
 			#endif
 				                    ;
-				// TODO: Get default sample rate
+				/* TODO: Get default sample rate */
 				want.mSampleRate = (float) *sample_rate;
-				// 16 bit output
+				/* 16 bit output */
 				want.mBitsPerChannel = sizeof(short) * 8;
 				want.mChannelsPerFrame = CLOWNAUDIO_STREAM_CHANNEL_COUNT;
-				// kAudioFormatLinearPCM doesn't use packets
+				/* kAudioFormatLinearPCM doesn't use packets */
 				want.mFramesPerPacket = 1;
-				// Bytes per channel * channels per frame
+				/* Bytes per channel * channels per frame */
 				want.mBytesPerFrame = (want.mBitsPerChannel / 8) * want.mChannelsPerFrame;
-				// Bytes per frame * frames per packet
+				/* Bytes per frame * frames per packet */
 				want.mBytesPerPacket = want.mBytesPerFrame * want.mFramesPerPacket;
 
 				error = AudioUnitSetProperty(stream->audio_unit, kAudioUnitProperty_StreamFormat, kAudioUnitScope_Input, 0, &want, sizeof(AudioStreamBasicDescription));
 
 				if (!error)
 				{
-					// Set the AudioUnit output callback
+					/* Set the AudioUnit output callback */
 					AURenderCallbackStruct callback;
-					// Callback function
+					/* Callback function */
 					callback.inputProc = Callback;
-					// User data
+					/* User data */
 					callback.inputProcRefCon = stream;
 
 					error = AudioUnitSetProperty(stream->audio_unit, kAudioUnitProperty_SetRenderCallback, kAudioUnitScope_Input, 0, &callback, sizeof(AURenderCallbackStruct));
 
 					if (!error)
 					{
-						// Initialize the configured AudioUnit instance
+						/* Initialize the configured AudioUnit instance */
 						error = AudioUnitInitialize(stream->audio_unit);
 
 						if (!error)

--- a/src/playback/coreaudio.c
+++ b/src/playback/coreaudio.c
@@ -1,5 +1,5 @@
 /*
-Copyright (c) 2018-2023 Clownacy
+Copyright (c) 2018-2023 Ned Loynd
 
 Permission to use, copy, modify, and/or distribute this software for any
 purpose with or without fee is hereby granted.

--- a/src/playback/coreaudio.c
+++ b/src/playback/coreaudio.c
@@ -1,20 +1,17 @@
-// Copyright (c) 2018-2023 Clownacy
-//
-// This software is provided 'as-is', without any express or implied
-// warranty.  In no event will the authors be held liable for any damages
-// arising from the use of this software.
-//
-// Permission is granted to anyone to use this software for any purpose,
-// including commercial applications, and to alter it and redistribute it
-// freely, subject to the following restrictions:
-//
-// 1. The origin of this software must not be misrepresented; you must not
-//    claim that you wrote the original software. If you use this software
-//    in a product, an acknowledgment in the product documentation would be
-//    appreciated but is not required.
-// 2. Altered source versions must be plainly marked as such, and must not be
-//    misrepresented as being the original software.
-// 3. This notice may not be removed or altered from any source distribution.
+/*
+Copyright (c) 2018-2023 Clownacy
+
+Permission to use, copy, modify, and/or distribute this software for any
+purpose with or without fee is hereby granted.
+
+THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+PERFORMANCE OF THIS SOFTWARE.
+*/
 
 #include "clownaudio/playback.h"
 

